tests/unit/amo/components/TestCategories.js

```diff
@@ -70,14 +70,14 @@ describe('<Categories />', () => {
       result: [
         {
           application: 'android',
-          name: 'Games',
-          slug: 'Games',
+          name: 'Travel',
```

Comment:
Ok. Should I add this test in components side or inside reducers?  Actually it expects the categoryResponse to be already sorted.